both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
+    return wca.roundByCode[this.roundCode];
+  },
+  eventName: function() {
+    return this.eventCode ? wca.eventByCode[this.eventCode].name : '';
```

Comment:
Yeah, moving them to their own table seems like a reasonable thing to do.

If this method shouldn't be called, then I'd rather it barf instead of silently returning something. Lets `assert(this.eventCode)`.